### ISSUE TITLE
Consistency during a connection

### ISSUE THREAD

**Issue by Acconut** [ietf-wg-httpapi/ratelimit-headers#140]
Draft -08 includes the following statement (emphasis mine):

> The "RateLimit-Policy" response header field is a non-empty List of Quota Policy Items ([Section 3.1](https://www.ietf.org/archive/id/draft-ietf-httpapi-ratelimit-headers-08.html#quotapolicy-item)). The Item value MUST be a String. Its value is informative. **The field value is expected to remain consistent over a the lifetime of a connection.** It is this characteristic that differentiates it from the [RateLimit](https://www.ietf.org/archive/id/draft-ietf-httpapi-ratelimit-headers-08.html#ratelimit-field) ([Section 4](https://www.ietf.org/archive/id/draft-ietf-httpapi-ratelimit-headers-08.html#ratelimit-field)) field that contains information that may change on every request.

Is _connection_ the correct term? I believe the intention is to state that the policies should not change between requests. Maybe we can rephrase it to express that better? How about

> The field value is expected to remain consistent across multiple requests.

?

**Comment by AviDuda** [comment]
I disagree with these two sentences entirely and think they should be dropped.

The current and also suggested wording would make it impossible to change the policy either with new server deployments or dynamically, as the client would just try to keep requesting more data, and would consider them as ongoing requests which should keep the same policy.

To make this work, this would require servers to keep track of clients and their current rate policies to make sure they don't change the value between requests, which seems unnecessarily complex.

It should be client's responsibility to honor any set rate limits, even if they change during requests.

**Comment by darrelmiller** [comment]
@AviDuda   The wording does not say the RateLimit-Policy field value MUST NOT change.  The server absolutely can change the the policy whenever it sees fit.  However, the value of the separate policy header is separate the header information that will definitely change every request from that which probably will not.

@Acconut   I also do not like our use of the word connection here. Maybe we can say something like "The field value is likely to remain constant within a sequence of requests from a client"

**Comment by darrelmiller** [comment]
@Acconut  I have adopted the "SHOULD" language but added more explanation as to why to address @AviDuda concerns.

> The field value SHOULD remain consistent over a sequence of HTTP responses. It is this characteristic that differentiates it from the [RateLimit](#ratelimit-field) field that contains information that MAY change on every request. The "RateLimit-Policy" field enables clients to control their own flow of requests based on policy information provided by the server. Situations where throttling constraints are highly dynamic are better served using the (RateLimit field)[{#ratelimit-field}] that communicates the latest service information a client can react to. Both fields can be communicated by the server when appropriate.

Even with the term SHOULD, clients still need to account for policies that change between requests.
